src/main/java/org/myrobotlab/service/Hd44780.java

```diff
@@ -196,97 +193,288 @@ public void display(String string, int line) {
     broadcastState();
     switch (line) {
       case 0:
-        lcdWrite((byte) 0x80);
+      setDDRAMaddress((byte) 0x00);
         break;
       case 1:
-        lcdWrite((byte) 0xC0);
+      setDDRAMaddress((byte) 0x40);
         break;
       case 2:
-        lcdWrite((byte) 0x94);
+      setDDRAMaddress((byte) 0x14);
         break;
       case 3:
-        lcdWrite((byte) 0xD4);
+      setDDRAMaddress((byte) 0x3C);

```

Comment:
The convention we follow with the myrobotlab is camel back case including abbreviations
So setDDRAMaddress should be setDdramAddress 
just like setXML would be setXml